### PR TITLE
feat: home page updates, live oracle rates, and indexer token fallback

### PR DESCRIPTION
**Pull request by chapati23**
### UI Updates
- Rename "Source" column to "Type" in pool tables
- Fix Monad pools not showing FPMM source badge
- Remove "Mainnet" suffix from chain labels (e.g. "Celo Mainnet" → "Celo")
- Strip trailing `.0`/`.00` decimals from formatted USD values
- Remove redundant Rebalancer column from pool tables
- Show partial LP count instead of N/A when some chains' LP queries fail
- Improve sub-KPI contrast and sizing
- Shorten volume column headers to "Vol."

### Live Oracle Rates (replaces hardcoded FX_RATES)
- `buildOracleRateMap()` derives USD rates from pool oracle prices at runtime
- All non-USDm pools (e.g. axlEUROC/EURm) now show volume in USD
- Legacy `cEUR` symbol aliased to `EURm` rate for historical fee data
- Rates refresh every 5 minutes with pool data — no code changes needed when exchange rates shift

### Indexer Token Fallback
- `KNOWN_TOKEN_META` built dynamically from `@mento-protocol/contracts` v0.4.0 (includes decimals)
- Filters out internal deployment names (StableToken*, Mock*)
- New tokens automatically picked up on package upgrades

## Test plan

- [x] 633 dashboard tests pass (40 test files)
- [x] 125 indexer tests pass
- [x] 11 new tests for `buildOracleRateMap` (token ordering, oracleOk guards, legacy alias, undefined handling)
- [x] FX-rate volume tests for both token0 and token1 paths
- [x] LP KPI edge cases (all fail → N/A, some fail → partial, 0 LPs + fail → "0")
- [x] `buildKnownTokenMeta` fallback test
- [x] trunk fmt + trunk check clean
- [ ] Visual verification of homepage KPI tiles and pool table

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: ui-dashboard/src/lib/tokens.ts [hunk 1]
```diff
@@ -6,8 +6,74 @@ import { truncateAddress, parseWei } from "./format";
...
+
+/** Maps token symbol → USD-per-1-token rate, derived from pool oracle prices. */
+export type OracleRateMap = Map<string, number>;
+
+/**
+ * Builds a symbol→USD rate map from pools that have a USDm leg.
+ * For each pool with one USDm token and a valid oraclePrice,
+ * extracts the USD rate for the non-USDm token.
+ */
+export function buildOracleRateMap(
```
Comment (chapati23) [inline ui-dashboard/src/lib/tokens.ts:31]: All cursor bot comments have been addressed in commits 3218bc0 and 2330858:

- **AUSD decimals**: Fixed from 18 → 6
- **LP KPI 0 vs N/A**: Shows N/A when all chains fail, partial count when some succeed
- **useMemo deps**: `rates` added to dependency array
- **buildOracleRateMap tests**: 9 unit tests added
- **cEUR legacy alias**: Added cEUR→EURm alias so historical fees remain priced
- **Integration test**: buildOracleRateMap is individually well-tested; full pipeline integration test deferred as lower priority

File: ui-dashboard/src/app/page.tsx [hunk 2]
```diff
@@ -154,12 +158,18 @@ function GlobalContent() {
           if (fees.isTruncated) isTruncated = true;
         }
 
-        // LP count
+        // LP count — accumulate from successful chains
         if (netData.uniqueLpCount !== null && totalUniqueLps !== null) {
           totalUniqueLps += netData.uniqueLpCount;
+          hasSuccessfulLpResult = true;
         }
       }
```
Comment (chapati23) [inline ui-dashboard/src/app/page.tsx:167]: Fixed in 7b320df — now tracks `hasSuccessfulLpResult` explicitly. A chain with genuinely 0 LPs + a failed chain shows "0" with partial subtitle, not N/A.
